docs/javascript/v2/features/change-role.mdx

```diff
@@ -33,20 +33,45 @@ console.log("can I end change role - ", permissions.changeRole);
 const { video, audio, screen } = hmsStore.getState(selectIsAllowedToPublish);
 ```
 
-## Change Role API
+## Change Role APIs
+
+There are two way of chaning peer's role using our api:
+1. [`Single Peer Role API`](#change-single-peer-role-api) - It is useful when we need to change role of a single peer.
+2. [`Bulk Peer Role API`](#change-bulk-peer-role-api) - We could change a role of all the peer to another role. It will be helpful to make breakout rooms, waiting rooms, etc.
+
+### Change Single Peer Role API
 
 If the local peer has permissions to change role(`permissions.changeRole`), they can change either their or someone else's role using
 the below interface.
 
 ```js
-hmsActions.changeRole(forPeerId, toRoleName, force);
+hmsActions.changeRoleOfPeer(forPeerId, toRoleName, force);
 ```
 
 - `forPeerId`: The peer ID whose role needs to be changed, the peer can be either of local or remote.
 - `toRoleName`: The target role name.
 - `force`(optional): The other peer gets a request by default to accept a role change, by setting force to true, the role can be changed
   without requesting for approval.
 
+
+### Change Bulk Peer Role API
+
+If the local peer has permissions to change role(`permissions.changeRole`), they can convert all roles from a list of roles, to another role using
+the below interface.
+
+```js
```

Comment:
we should have both react and js snippets even though this is straightforward for us.

Reply:
Do we need to add react snippet in line 63? @Nixon-Samuel

Reply:
yes @amar-1995 Would be easy for customers to adopt.

Reply:
done